qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -90,10 +92,20 @@ def __init__(
                     "num_bootstrap must be None or an integer greater than or equal to 2"
                 )
 
-        if isinstance(self._gss.solver, VariationalAlgorithm):  # type: ignore
+        if isinstance(self._gss, GroundStateSolver) and isinstance(
+            self._gss.solver, VariationalAlgorithm
+        ):  # type: ignore
             # Save initial point passed to min_eigensolver;
             # this will be used when NOT bootstrapping
             self._initial_point = self._gss.solver.initial_point  # type: ignore
+        elif (
+            isinstance(self._gss, ExcitedStatesSolver)
+            and hasattr(self._gss, "_gsc")
+            and isinstance(self._gss._gsc.solver, VariationalAlgorithm)
+        ):  # type: ignore
+            # Save initial point passed to min_eigensolver;
+            # this will be used when NOT bootstrapping
+            self._initial_point = self._gss._gsc.solver.initial_point  # type: ignore
```

Comment:
Might we even expose the `initial_point` setter of the GSS on the ESS?

Reply:
Does not seem needed anymore after having exposed the solver